package.json

```diff
@@ -2,7 +2,7 @@
   "dependencies": {
     "browser-ui-test": "^0.21.1",
     "es-check": "^6.2.1",
-    "eslint": "^8.57.1",
+    "eslint": "^8.6.0",
```

Comment:
The version got downgraded?

Reply:
That was a mistake made during rebase. I'll fix it.